src/__test__/__mock__/mockHTML.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable import/prefer-default-export */
```

Comment:
[OPTIONAL]

- I think it is missed by the previous reviewer because of this I'm making this optional but it's highly recommended. We always tell students to improve the code instead of disabling the linter, all the rules are for you and to make you a better programmer. :smile: